src/components/Book.js

```diff
@@ -3,40 +3,66 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import Button from './Button';
 import { removeBook, removeAsyncBook } from '../redux/books/booksSlice';
+import './styles/book.css';
 
-function Book({ id, title, author }) {
+function Book({
+  id, title, author, category,
+}) {
   const dispatch = useDispatch();
   const handelRemoveBook = () => {
     dispatch(removeAsyncBook(id));
     dispatch(removeBook(id));
   };
   return (
-    <>
-      <div>
-        <ul>
+    <div className="Lesson-Panel">
+      <p className="school-of">
+        {category}
+        {' '}
+      </p>
+      <h4 className="title">{title}</h4>
+      <p className="suzanne-collins">{author}</p>
+      <div className="flex-spac-btwn">
+        <div className="buttons-contanier flex">
+          <Button className="comments-btn" value="Comment" type="button" />
+          <hr />
+          <Button className="remove-btn" type="button" onClick={handelRemoveBook} value="Remove" />
+          <hr />
+          <Button className="edit-btn" value="Edit" type="button" />
+        </div>
+        <div className="progress-chapter  flex-spac-btwn">
+          <div className="percent-container flex ">
+            <div className="oval-2" />
+            <div className="percent-info">
+              <p className="percent-complete">100%</p>
+              <p className="completed">Completed</p>
+            </div>
+          </div>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly add the vertical gray line on each book that is on the design. You currently dont have the gray ie

Here is the line being referred to 👇🏽
![image](https://github.com/naji-97/sec-book-store/assets/63473584/99c34c85-f22e-4afd-af70-6c91fc27c92b)